app/controllers/nonsipp/landorproperty/LandOrPropertyTotalIncomeController.scala

```diff
@@ -58,7 +58,7 @@ class LandOrPropertyTotalIncomeController @Inject()(
     implicit request =>
       request.userAnswers.get(LandOrPropertyChosenAddressPage(srn, index)).getOrRecoverJourney { address =>
         val preparedForm = request.userAnswers.fillForm(LandOrPropertyTotalIncomePage(srn, index), form)
-        Ok(view(viewModel(srn, index, preparedForm, address.addressLine1, mode)))
+        Ok(view(preparedForm, viewModel(srn, index, form, address.addressLine1, mode)))
```

Comment:
Same question as previously for a slightly different scenario: should `form` parameter value of viewModel be `preparedForm` here and in similar code changes?

Reply:
No, for the same reason as above:

"No, this is intentional - the `form` parameter in the `viewModel` method call is used only to satisfy further method calls and constructors, as it needs the form type to do so (e.g. in `FormPageViewModel` and `SingleQuestion`). The `form` parameter in the `view` method call is what's used to populate the page with any pre-populated data."